packages/features/ee/organizations/pages/settings/appearance.tsx

```diff
@@ -43,6 +43,12 @@ const OrgAppearanceView = ({
   } = themeForm;
 
   const [hideBrandingValue, setHideBrandingValue] = useState(currentOrg?.hideBranding ?? false);
+  const [allowSEOIndexingValue, setAllowSEOIndexingValue] = useState(
```

Comment:
Based on the screenshot, we want this on the profile page of organization settings and not appearance. Could also be on the general page of org settings as that is where it is for normal users.